<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefScreenshots</class>
 <widget class="QWidget" name="PrefScreenshots">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Screenshots</string>
  </property>
  <property name="windowIcon">
   <iconset theme="accessories-screenshot-tool"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="screenshot_template_label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Template:</string>
     </property>
     <property name="buddy">
      <cstring>screenshot_template_edit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="screenshot_format_label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Format:</string>
     </property>
     <property name="buddy">
      <cstring>screenshot_format_combo</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="screenshot_template_edit">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="screenshots_dir_label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Folder:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>screenshot_edit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QComboBox" name="screenshot_format_combo">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>jpg</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>png</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>webp</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="screenshot_edit">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons/res.qrc"/>
 </resources>
 <connections/>
</ui>
